ts/src/main/scala/ch/epfl/ts/indicators/RangeIndicator.scala

```diff
@@ -0,0 +1,77 @@
+package ch.epfl.ts.indicators
+
+import ch.epfl.ts.component.Component
+import ch.epfl.ts.data.OHLC
+import scala.collection.mutable.MutableList
+import com.sun.xml.internal.bind.v2.model.annotation.Init
+
+class RI(val support : Double, val resistance : Double, val period : Int)
+case class RI2( override val support : Double, override val resistance : Double, override val period : Int ) extends RI(support,resistance,period)
```

Comment:
What is the difference between `RI` and `RI2`? If you really need both, rename them to something more explicit + write JavaDoc to explain.